tests/testthat/test-vignette.R

```diff
@@ -37,3 +37,10 @@ test_that("use_vignette() does the promised setup", {
     c(VignetteBuilder = "knitr")
   )
 })
+
+test_that("use_article goes in article subdirectory", {
+  create_local_package()
+
+  use_article("test")
+  expect_true(file_exists(proj_path("vignettes/articles/test.Rmd")))
+})
```

Comment:
This comes up so much we have a helper for it:

https://github.com/r-lib/usethis/blob/master/tests/testthat/helper.R#L131